manual/src/adding_a_parser.md

```diff
@@ -93,10 +93,13 @@ language.
 "json" => Some(Json),
 ```
 
-There may also file names or shebangs associated with your
-language. [GitHub's linguist
+There may also file names or shebangs associated with your language. Add those by changing the `LANG_EXTENSIONS` constant further below in that file. GitHub's linguist definitions is a useful source of common file extensions.
+
+
+There may also file names or shebangs associated with your language.
```

Comment:
Looks like you've duplicated a sentence here?

Reply:
Whoops, indeed. Fixed.